ietf-optical-impairment-topology.tree

```diff
@@ -59,29 +67,32 @@ module: ietf-optical-impairment-topology
     |        +--ro OTSi-signal-width?        decimal64
     |        +--ro channel-delta-power?      decimal64
     +--ro transponders-list* [transponder-id]
-       +--ro transponder-id                      uint32
-       +--ro (mode)?
-       |  +--:(G.692.2)
-       |  |  +--ro standard-mode?                standard-mode
-       |  +--:(organizational-mode)
-       |  |  +--ro operational-mode?             operational-mode
-       |  |  +--ro organization-identifier?      vendor-identifier
-       |  +--:(explicit-mode)
-       |     +--ro available-modulation-types*   identityref
-       |     +--ro configured-modulation-type?   identityref
-       |     +--ro available-baud-rates*         uint32
-       |     +--ro configured-baud-rate?         uint32
-       |     +--ro available-FEC-types*          identityref
-       |     +--ro configured-FEC-type?          identityref
-       |     +--ro FEC-code-rate?                decimal64
-       |     +--ro FEC-threshold?                decimal64
-       +--ro power?                              int32
-       +--ro power-min?                          int32
-       +--ro power-max?                          int32
-  augment /nw:networks/nw:network/nw:node/tet:te/tet:tunnel-termination-point:
+       +--ro transponder-id      uint32
+       +--ro tranceivers-list* [tranceiver-id]
+          +--ro tranceiver-id      uint32
+          +--ro supported-modes* [mode-id]
+          |  +--ro mode-id                          string
+          |  +--ro (mode)
+          |     +--:(G.692.2)
+          |     |  +--ro standard-mode?             standard-mode
+          |     +--:(organizational-mode)
+          |     |  +--ro operational-mode?          operational-mode
+          |     |  +--ro organization-identifier?   vendor-identifier
+          |     +--:(explicit-mode)
+          |        +--ro supported-std-modes
+          |           +--ro supported-application-codes*
+          |           |       -> ../../mode-id
+          |           +--ro supported-organizational-modes*
+          |                   -> ../../mode-id
+          +--ro power?             int32
```

Comment:
Current power values and min/max power values are needed for both directions separately: Tx and Rx

Reply:
The new grouping common-transceiver-capabilities-organizational-explicit in layer0-types-ext contain both power values min/max power values for both directions separately Tx and Rx. I suggest to delete "power" values and add the correct groupings as soon as layer0-types-ext module has been completely revised.

Reply:
Resolved with 4cb3d220117948e9912e40a00b320e2d32a58797